### ISSUE TITLE
SwiftLint 0.54.0 not available through homebrew

### ISSUE THREAD

**Issue by mikaelbartlett** [realm/SwiftLint#5353]
### New Issue Checklist

Not possible
~~- [ ] Updated SwiftLint to the latest version~~
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

When installing or upgrading swiftlint through homebrew v0.53.0 is installed and not v0.54.0
Tried on two different computers.

**Comment by jpsim** [comment]
Correct, SwiftLint 0.54.0 is not on homebrew.

Please read the discussion in https://github.com/Homebrew/homebrew-core/pull/153931 for details.

**Comment by mplorentz** [comment]
If this is a problem for you because your Github Actions CI is pinning the latest version of swiftlint, and now CI is reporting errors that your local swiftlint does not you can use this Github Actions yaml to pin version 0.53.0:
```
   steps:
      - uses: actions/checkout@v1
      - name: SwiftLint
        uses: docker://norionomura/swiftlint:0.53.0
        with:
          args: swiftlint --strict --reporter github-actions-logging
```

**Comment by SimplyDanny** [comment]
SwiftLint 0.55.1 is available on Homebrew now.
